package/itest/ubuntu.sh

```diff
@@ -29,7 +29,8 @@ export TZ=US/Pacific
 ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
 apt-get update && apt-get install -y software-properties-common
-apt-get install -y --force-yes python3.7 python3-pip python3-yaml awscli
+add-apt-repository ppa:deadsnakes/ppa
+apt-get install -y --force-yes python3.7 python3-pip python3-yaml awscli python3.7-distutils
```

Comment:
not really sure we i had to add distutils here - afaik, this should have been installed as part of my debian/{rules,control} change

(the rest of these should also probably be in debian/control too so that they'd be automatically installed, probably)